aguskoll/target-app
# Create target

### What's in

View for crating the target
input validation
Post to the backend to create the target

### Todo

Show the topics list

### How it looks:

https://drive.google.com/file/d/1plJ74jqwIy7dNGMpkXzDguMcdohMjZRO/view?usp=sharing

![Screenshot_1611580786](https://user-images.githubusercontent.com/12178777/105712735-fb8aa580-5ef8-11eb-953b-c1c19d3f4707.png)

![Screenshot_1611581048](https://user-images.githubusercontent.com/12178777/105712747-00e7f000-5ef9-11eb-82f5-b5981fd15d38.png)

### How to test

Run this branch and log in.
Touch the create a target btn and fill data with topic 2
verify that you see a toast with "success creating target".
